hypershift-operator/metrics_test.go

```diff
@@ -137,3 +141,100 @@ func TestMetrics(t *testing.T) {
 		})
 	}
 }
+
+func TestOperatorInfo(t *testing.T) {
```

Comment:
this should be called TestGetOperatorImage and test exactly that function input/output